client/src/components/App.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import axios from 'axios';
-import Review from './Review';
+import Review from './review';
```

Comment:
Is there a reason this isn't capitalized like the other files?

Reply:
@Pfdiamond13 it's being really weird locally. The file ./review won't change to ./Review whenever I try to modify it. I'll try to look into it more.